examples/docker-for-mac/my_test.sh

```diff
@@ -65,17 +66,37 @@ configure_kubectl() {
     local port
     port=$(get_apiserver_arg "$apiserver_id" --secure-port)
 
-    docker cp "$HOME/.kube" "$testcontainer_id:/root/.kube"
-    docker exec "$testcontainer_id" kubectl config set-cluster docker-desktop "--server=https://$ip:$port"
-    docker exec "$testcontainer_id" kubectl config set-cluster docker-desktop --insecure-skip-tls-verify=true
-    docker exec "$testcontainer_id" kubectl config use-context docker-desktop
+    docker cp "$HOME/.kube" ct:/root/.kube
+    docker_exec kubectl config set-cluster docker-desktop "--server=https://$ip:$port"
+    docker_exec kubectl config set-cluster docker-desktop --insecure-skip-tls-verify=true
+    docker_exec kubectl config use-context docker-desktop
 }
 
-run_test() {
-    git remote add k8s https://github.com/helm/charts.git &> /dev/null || true
-    git fetch k8s
-    docker exec "$testcontainer_id" ct lint --chart-dirs stable,incubator --remote k8s
-    docker exec "$testcontainer_id" ct install --chart-dirs stable,incubator --remote k8s
+install_tiller() {
+    echo 'Installing Tiller...'
+    docker_exec kubectl --namespace kube-system create sa tiller
+    docker_exec kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+    docker_exec helm init --service-account tiller --upgrade --wait
+    echo
+}
+
+install_charts() {
+    echo "Add git remote k8s ${CHARTS_REPO}"
+    git remote add k8s "${CHARTS_REPO}" &> /dev/null || true
```

Comment:
`CHARTS_REPO` is not defined. Why change this? Maybe we shjould emove the separate remote which is just confusing sand use `origin` which is the default.